ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-periodics.yaml

```diff
@@ -383,3 +383,56 @@ periodics:
     - configMap:
         name: prow-job-cluster-launch-e2e-azure-conformance
       name: job-definition
+- agent: kubernetes
+  cron: ""
+  decorate: true
+  interval: 24h
+  name: periodic-ci-azure-bushslicer
+  skip_cloning: true
+  spec:
+    containers:
+    - args:
+      - --git-ref=openshift/openshift-azure@master
+      - --secret-dir=/usr/local/e2e-secrets/azure
+      - --target=e2e-azure
+      - --template=/usr/local/e2e-azure
+      command:
+      - ci-operator
+      env:
+      - name: CLUSTER_TYPE
+        value: azure
+      - name: CONFIG_SPEC
+        valueFrom:
+          configMapKeyRef:
+            key: openshift-openshift-azure-master.yaml
+            name: ci-operator-configs
+      - name: JOB_NAME_SAFE
+        value: e2e-azure
+      - name: TEST_IMAGE
+        value: quay.io/kwoodson/bushslicer
```

Comment:
this image "weighs" 2.1G. thats huuugeee!!

Reply:
Wanna try docker pull weblogic? :D

Reply:
@cjryan, I believe this could be reduced if we base the image off of centos and remove java and jenkins dependencies.  This would also require yet another dockerfile.  WDYT?

Reply:
@kwoodson yeah, I agree. The issue here is the existing dockerfile started life as a jenkins node worker; since this is a ci-operator job, we could probably eliminate a lot  of the existing filesize. Also suggested this here: https://github.com/openshift/BushSlicer/pull/30#issuecomment-443021606 will see if we can get a quorum with @akostadinov

Reply:
I have already done that in [openshift/verification-tests#30](https://github.com/openshift/verification-tests/pull/30). This is image can be used in our openshift-ci. The last think that we need to determine is if all the dependencies that are currently being installed [here](https://github.com/openshift/verification-tests/tree/master/tools/os_deps), are needed.

Reply:
@droslean, When your PR merges and gets verification-tests repository built then we can switch to using that image.

Reply:
you can use the value `registry.svc.ci.openshift.org/ci/bushslicer:centos` or configure the ci-operator config `build_root` to use that image instead.

Reply:
@droslean, thanks!